README.md

```diff
@@ -22,11 +22,11 @@ A proposed [WebAssembly System Interface](https://github.com/WebAssembly/WASI) A
 
 ### Introduction
 
-The `wasi-runtime-config` world aim to provide a set of generic interfaces for dynamic configuration management systems. They are often the source of the configuration truth that lives in external services. Configuration values are often polled by the application for
+The `wasi-runtime-config` world aim to provide a set of generic interfaces for providing configuration to a component. Configuration values are often polled by the application for
 
-1. toggling on/off feature flags
-2. A/B testing
-3. reading user allow/deny lists
+1. Configuring the runtime behavior of a component
```

Comment:
runtime behavior of a component seems to be a bit too vague in the sense that runtime behvaior is a vastly general term. Could you break it down to what it means for a component to configure its runtime behavior?

Reply:
I feel like that might get down into "splitting hairs" territory. It is very general, but purposefully so. Some applications are fairly crazy and run an entirely different code path depending on config values passed in. Others use it to configure something like a port or URL to connect to. Others may do things like set concurrency values. I could keep enumerating those, but I am not sure that would be valuable unless we can't agree on what "configuration" is

Reply:
Now that I am thinking about it more, I'll add a few examples of what is meant by config